includes/Classifai/Providers/OpenAI/Images.php

```diff
@@ -9,30 +9,36 @@
 use Classifai\Providers\Provider;
 use Classifai\Providers\OpenAI\APIRequest;
 use WP_Error;
-use WP_REST_Server;
 
-class DallE extends Provider {
+class Images extends Provider {
```

Comment:
I am changing the class name here but don't think we need to worry about deprecating this as it's unlikely anyone was directly extending this class